modules/ocf_mirrors/manifests/ftpsync.pp

```diff
@@ -42,11 +43,23 @@
     require => File[$project_path];
   }
 
-  cron { "ftpsync-${title}":
-    command => "BASEDIR=${project_path} ${project_path}/bin/ftpsync > /dev/null 2>&1",
-    user    => 'mirrors',
-    minute  => $cron_minute,
-    hour    => $cron_hour,
-    require => File["${project_path}/bin", "${project_path}/etc/ftpsync.conf"];
+  # TODO: Remove once systemd timers have been tested for reliability
+  if $use_systemd {
+    ocf_mirrors::timer { "ftpsync-${title}":
+      exec_start   => "${project_path}/bin/ftpsync > /dev/null 2>&1",
+      environments => { 'BASEDIR' => $project_path },
+      hour         => $cron_hour,
+      minute       => $cron_minute,
+      require      => File["${project_path}/bin", "${project_path}/etc/ftpsync.conf"],
+    }
+  }
+  else {
```

Comment:
nit: Put the previous `}`and this `else {` on the same line (like `} else {`), mostly just to follow the convention in this repo so far.